src/cargo/ops/registry.rs

```diff
@@ -972,18 +974,21 @@ pub fn registry_logout(config: &Config, reg: Option<&str>) -> CargoResult<()> {
 }
 
 pub struct OwnersOptions {
-    pub krate: Option<String>,
     pub token: Option<Secret<String>>,
     pub index: Option<String>,
-    pub to_add: Option<Vec<String>>,
-    pub to_remove: Option<Vec<String>>,
-    pub list: bool,
+    pub subcommand: Option<String>,
+    pub subcommand_arg: Option<ArgMatches>,
```

Comment:
This layer of the API should not involve `clap`

The simplest option is to just keep the old API

Reply:
Thanks for your help, I've reinstated the structure. However, for the new subcommand, two elements have to be added.